test/unit/util.test.ts

```diff
@@ -65,3 +67,45 @@ test('Get full URL', async(t) => {
             'https://wikimedia.org/api/rest_v1/media/math/render/svg/34cbb1e27dae0c04fc794a91f2aa001aca7054c1',
             'Full Url when base and url both strtas with http/s');
 })
+
+test('Custom flavour path', async(t) => {
+
+    // checks in current working directory.
+    let createStream = fs.createWriteStream(pathParser.resolve(process.cwd(), 'testCustomFlavour.js'));
+
+    t.equal(getCustomFlavorPath('testCustomFlavour.js'), pathParser.resolve(process.cwd(), 'testCustomFlavour.js'),
+        'Custom flavour in working directory.');
+
+    // checks when current and working directory have file with same name(preference to working directory).
+    createStream = fs.createWriteStream(pathParser.resolve(process.cwd(), 'extensions/testCustomFlavour.js'));
+
+    t.equal(getCustomFlavorPath('testCustomFlavour'), pathParser.resolve(process.cwd(), 'testCustomFlavour.js'),
+        'When file with same name exist in extensions directory.');
+
+    createStream.end();
+    fs.unlinkSync(pathParser.resolve(process.cwd(), 'testCustomFlavour.js'));
+    fs.unlinkSync(pathParser.resolve(process.cwd(), 'extensions/testCustomFlavour.js'));
+
+    // checks in extension directory.
+    t.equal(getCustomFlavorPath('wiktionary_fr.js'), pathParser.resolve(process.cwd(), 'extensions/wiktionary_fr.js'),
+        'Custom flavour in extensions directory.');
+
+    t.equal(getCustomFlavorPath('wiktionary_fr'), pathParser.resolve(process.cwd(), 'extensions/wiktionary_fr.js'),
+        'Custom flavour in extension directory without js extension.');
+
+    // checks in absolute path.
+    createStream = fs.createWriteStream(pathParser.resolve(process.cwd(), 'extensions/testCustomFlavour.js'));
```

Comment:
You should not use `process.cwd()` here because you have no clue where the test is started from. Secure the path by changing the current directory for the test which test a local/same_dir `*.js` file... Otherwise you need to build the paths related to something which is fix, like the root of the codebase. You made a similar error by reading the files for the Hero polyfill.